seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -126,6 +126,14 @@ public int getBytesSize(SeaTunnelRowType rowType) {
         return size;
     }
 
+    public boolean isBinaryFormat() {
+        return getOptions().getOrDefault("is_binary_format", false).equals(true);
+    }
+
+    public boolean isComplete() {
+        return getOptions().getOrDefault("is_complete", false).equals(true);
+    }
```

Comment:
The option value belong binary format, so I think we should not put the method into `SeaTunnelRow`.